Initial plugin to modify materials and short Mk2 fuselage normal map

blowfishpro opened the pull request:
More normal maps to come but wanted to get plugin and an initial example in first

ChrisAdderley commented on Distribution/Restock/GameData/ReStock/Patches/FuelTank/restock-fueltanks-mk2.cfg:
```diff
@@ -0,0 +1,18 @@
+@PART[mk2FuselageShortLiquid|mk2FuselageShortLFO|mk2FuselageShortMono]:FINAL
+{
+    MODULE
```
Can you add attribution (to yourself) for the content?

blowfishpro replied:
Yeah, I'll do that.  Also forgot to make this patch non-final.

ChrisAdderley commented:
Looks good generally - just the one note about attribution.

blowfishpro commented:
Force pushed but the changes from the most recent commit are:
* Added a section at the top of the patch file explaining what's in it
* Made patch `:FOR[Restock]` rather than `:FINAL`
* Added myself to the author field of those parts
